docs/modules/ROOT/pages/access-control.adoc

```diff
@@ -202,3 +134,143 @@ TIP: A recommended configuration is to grant both roles to a secure governance c
 Operations executed by the xref:api:governance.adoc#TimelockController[`TimelockController`] are not subject to a fixed delay but rather a minimum delay. Some major updates might call for a longer delay. For example, if a delay of just a few days might be sufficient for users to audit a minting operation, it makes sense to use a delay of a few weeks, or even a few months, when scheduling a smart contract upgrade.
 
 The minimum delay (accessible through the xref:api:governance.adoc#TimelockController-getMinDelay--[`getMinDelay`] method) can be updated by calling the xref:api:governance.adoc#TimelockController-updateDelay-uint256-[`updateDelay`] function. Bear in mind that access to this function is only accessible by the timelock itself, meaning this maintenance operation has to go through the timelock itself.
+
+[[access-management]]
+== Access Management
+
+Although xref:api:access.adoc#AccessControl[`AccessControl`] offers a more dynamic solution for adding permissions to your contracts, decentralized protocols tend to become more complex after integrating new contract instances, requiring keeping track of permissions in each one. This increases the complexity of permissions management and monitoring across the system.
+
+image::access-control-multiple.svg[Access Control multiple]
+
+Protocols managing permissions in production systems often require more integrated alternatives to fragmented permissions through multiple `AccessControl` instances. 
+
+Better role management can be achieved with an xref:api:access.adoc#AccessManager[`AccessManager`] instance. Instead of managing each contract's permission separately, AccessManager stores all the permissions in a single contract, making your protocol easier to audit and maintain.
+
+image::access-manager.svg[Access Manager]
+
+The AccessManager is designed around the concept of role:
+
+* Roles are granted to accounts (addresses) following a many-to-many approach. This means that each user can have one or multiple roles and multiple users can have the same role.
```

Comment:
For what purpose? Flexibility?

Reply:
Yes. Specifically, the intention was to reduce as much as possible the amount of interactions required when modifying permissions and the most likely to change are the permissioned addresses.

Now that I mention it, we added Multicall to this contract for batched permission changes, shall we mention it?